.circleci/config.yml

```diff
@@ -54,10 +54,10 @@ jobs:
       node_version: '18.2'
     steps:
       - checkout
-      - rn/yarn_install:
-          yarn_install_directory: examples/purchaseTesterTypescript
-          cache_folder: ~/.cache/yarn
```

Comment:
`rn/yarn_install` wasn't installing yarn either, it was running `yarn install`. I believe yarn comes installed in the executor, and it will pick the version defined in `.yarn`